modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/itemselector/plugin.js

```diff
@@ -265,7 +265,13 @@
 						callback(imageSrc, selectedItem);
 					}
 					else {
-						let elementOuterHtml = '<img src="' + imageSrc + '">';
+						let elementOuterHtml =
+							'<img src="' +
+							imageSrc +
+							'" height="' +
+							editor.document.$.body.getBoundingClientRect()
+								.height +
+							'">';
```

Comment:
Can we use a template literal to make it a bit cleaner?

```js
let elementOuterHtml = `<img src="${imageSrc}" height="${editor.document.$.body.getBoundingClientRect().height}">`;
```

Also, what is this syntax? `editor.document.$`

Reply:
Made the changes and pushed them